src/styles/components/BlockTitleDescription.scss

```diff
@@ -0,0 +1,29 @@
+.BlockTitleDescription {
+  &:nth-of-type(even) {
+    background-color: color('light-gray');
+  }
+
+  &__container {
+    @include media('md-down') {
```

Comment:
i think we generally prefer mobile first for css, is it possible to set `max-width: 35rem` for mobile and use `md-up` to set max-width to auto instead?